### PR TITLE
Feedback from Coralie

### PR DESCRIPTION
**Pull request by m-maillot**
Feedback from Coralie

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,23 +50,28 @@
 ## Presentation
 ### The problems
```
Comment (yannhermouet) [inline README.md]: The Challenges

File: README.md [hunk 2]
```diff
@@ -50,23 +50,28 @@
...
 
-You are like us, a small team that build their own products and deliver a ton to different customers at different stages of the development cycle (Development, Beta Testing, Production, …) but have no unified platform to ship your work easily…
+#### Development agencies
```
Comment (yannhermouet) [inline README.md]: Development Agencies

File: README.md [hunk 3]
```diff
@@ -50,23 +50,28 @@
...
+
+You are the Mobility Head of a large company with fragmented teams in different geos. They all run their own infrastructure and development cycle based on the apps they ship, the technology choices they make and the business needs they have to answer... You have limited visibility on early stages development, no control on testing deployment and got notified only when apps are sent to production… TThe icing on the cake: it’s your responsibility when an app ships to a public store or your own EMM...
+Or any other hair pulling scenarios in between...
 The end of these hellish days is upon us, we've built Shuttle for you!
 
+
 ### The solution
 
-Shuttle is a product dedicated to build distribution.
+Shuttle is a product dedicated to manage distribution.
```
Comment (yannhermouet) [inline README.md]: Shuttle is a product dedicated to apps management and distribution.

File: README.md [hunk 4]
```diff
@@ -50,23 +50,28 @@
...
+#### Development agencies
 
-You lead the development team of a digital company with a single mobile product. The thing is, “single product” doesn’t mean “single version” ; between the production, testing, development and R&D, you find yourself maintaining dozens of different builds and no easy way to push them…
+You are like us, a small team that builds its own products and delivers a ton to different customers at different stages of the development cycle (Development, Beta Testing, Production,...) but has no suitable tool to ship your work easily…
 
-You are the Mobility Head of a large company with fragmented teams in different geos. They all run their own infrastructure and development cycle based on the apps they ship, the technology choices they make and the business needs they have to answer… You have limited visibility on early stages development, no control on testing deployment and get notified only when apps goes to production… The saddest thing is: it’s your responsibility when an app ship to a public store or your own EMM…
+#### For small teams
```
Comment (yannhermouet) [inline README.md]: For Digital Product Team

File: README.md [hunk 5]
```diff
@@ -50,23 +50,28 @@
...
 
-Or any other hair pulling scenarios in between…
+You lead the development team of a digital company with a single mobile product. But, “single product” doesn’t mean “single version”: between production, testing, development or R&D, etc., you end up maintaining dozens of different builds and no easy way to manage or push them…
 
+#### For large companies
```
Comment (yannhermouet) [inline README.md]: For Large Companies

**Comment by m-maillot** [comment]
@yannhermouet we can merge this PR and create a new one after with the new content ?

**Pull request merged**